### DISCUSSION TITLE
How to change the cursor when window is not in focus

### DISCUSSION THREAD

**Discussion by tembbo** [ghostty-org/ghostty#3542] [Q&A] [answered]
Is there any way of changing the cursor in ghostty when the window is not in focus (e.g. clicking outside the window).  An example of this would be the kitty option: https://sw.kovidgoyal.net/kitty/conf/#opt-kitty.cursor_shape_unfocused

**Comment by 00-kat** [comment, marked as answer]
P.S.: #2818.
